DashKit/DashPumpManager.swift

```diff
@@ -1795,8 +1814,7 @@ extension DashPumpManager: PodSDKLoggingShimDelegate {
 
 // MARK: - AlertResponder implementation
 extension DashPumpManager {
-    public func acknowledgeAlert(alertIdentifier: Alert.AlertIdentifier) {
-        
+    public func acknowledgeAlert(alertIdentifier: Alert.AlertIdentifier, completion: @escaping (Error?) -> Void) {
```

Comment:
Hm.  Is `DashPumpManager` actually an `AlertResponder`?  I don't see it declaring its conformance.  If it is, then this isn't going to build.  If it isn't, then, who calls this?

Reply:
DashPumpManager is a PumpManager, which conforms to DeviceManager, which conforms to AlertResponder

Reply:
Here's the LoopKit PR.  I thought I had created PRs for all of these, but somehow dropped that: https://github.com/tidepool-org/LoopKit/pull/387